firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -54,6 +55,13 @@ public List<Component<?>> getComponents() {
             .add(Dependency.requiredProvider(TransportFactory.class))
             .factory(FirebasePerfRegistrar::providesFirebasePerformance)
             .build(),
+        Component.builder(FirebasePerfEarly.class)
```

Comment:
should FirebasePerformance component take a dependency on FirebasePerfEarly to make sure it gets initialized before FirebasePerformance?

Reply:
Being eager I'm not sure it's strictly necessary, but it's good practice and trivial, so added.

Reply:
Note that in order to actually cause initialization you need to call `container.get(FirebasePerfEarly.class` inside providesFirebasePerformance, declaring the dep will not do it automatically as it's only used to detect dependency cycles